packages/web/project.json

```diff
@@ -15,7 +15,10 @@
         "baseHref": "/",
         "main": "packages/web/src/main.tsx",
         "tsConfig": "packages/web/tsconfig.app.json",
-        "assets": ["packages/web/src/favicon.ico", "packages/web/src/assets"],
+        "assets": [
+          "packages/web/src/favicon.ico",
+          { "input": "packages/web/src/public", "glob": "**/*", "output": "/" }
```

Comment:
This will copy translation files into the build. They will be loaded on the fly when they are needed so we don't bloat the bundle.